docs/tutorials/working-with-contexts.md

```
@@ -0,0 +1,440 @@
+```
+
+## Step 7: Clear Context (View Everything)
+
+To see all scopes without filtering:
+
+```bash
+scopes context current --clear
+```
+
+Output:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)